### PR TITLE
DR-146-design-landing-page

### PR DESCRIPTION
**Pull request by anquail**
### Summary of Changes

- The changes in this PR add responsive styling to the landing page to max the designs for desktop and mobile
---

### Summary of Implementation Logic

- updates copy and adds styling to necessary elements to match the design 
---

### Issue Link

Original Issue: https://github.com/RubySpeeders/Drum-Roulette/issues/146

Dependent Issues:

Depends on Issues:

---

### Screen shots / Videos

Screenshot 2023-12-18 at 3 54 29 PM
Screenshot 2023-12-18 at 3 54 06 PM

---

### Acceptance Criteria

 - [x] functionality remains (have all branches go to Navy musicians)
 - [x] match Figma designs
---

### Checklist for PR Owner

- [x] Standardized PR Title
- [x] Included relevant videos and screen shots
- [x] Included Summary of Changes
- [x] Included Summary of Implementation Logic
- [x] Acceptance Criteria are fufilled

### CODE REVIEW

File: client/src/components/landing.tsx [hunk 1]
```diff
@@ -2,59 +2,119 @@
...
+      <h2 className={classes.h2}>Hello!</h2>
+      <h3 className={classes.h3}>
+        Randomly generate band percussion assignments with just a few clicks.
+      </h3>
+      <div className={classes.h3Container}>
+        <h3 className={classes.h3}>Select Branch</h3>
+      </div>
+      <Grid container spacing={{ xs: 2, md: 3 }}>
+        {branches.map((branch) => (
+          <Grid key={branch.branch_id} item xs={6} md={3}>
```
Comment (RubySpeeders) [inline client/src/components/landing.tsx:90]: could we possibly pull this out into its own component?

File: client/src/app/page.tsx [hunk 2]
```diff
@@ -1,5 +1,8 @@
+import getBranches from "@/utils/api/getBranches";
 import Landing from "../components/landing";
 
-export default function Home() {
-  return <Landing />;
+export default async function Home() {
+  const branchesData = await getBranches();
```
Comment (RubySpeeders) [inline client/src/app/page.tsx:5]: woot!

File: client/src/components/landing.tsx (hunk 1 above)
Comment (anquail) [reply client/src/components/landing.tsx]: created a `branchesGrid` component. I could also pull each branch logo/link/caption into its own `branchLogo` component or similar - lmk!

**Review by RubySpeeders — APPROVED** [review]
looks great!

**Pull request merged**